source/apiVolontaria/volunteer/admin.py

```diff
@@ -88,6 +92,61 @@ class EventAdmin(admin.ModelAdmin):
     ordering = ('start_date',)
 
 
+class CycleAdmin(admin.ModelAdmin):
+
+    list_display = [
+        'name',
+        'start_date',
+        'end_date',
+    ]
+
+    date_hierarchy = 'start_date'
+
+    actions = ['generate_participation_report_csv', ]
+
+    def generate_participation_report_csv(self, request, queryset):
+        """
+        This function will take all the Participations generated from
+        this a cycle ang compile the time of every volunteer that has
+        participated
+        :param request: The request object of the admin
+        :param queryset: This is the admin panel queryset.
+        :return: This will return a CSV file
+        """
```

Comment:
You have some typos in the documentation :
 - `from this a cycle`
 - `cycle ang compile`

Reply:
Oups, corrected.